<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpMainDNS">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>171</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Main DNS Server</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="dnsGoogleMain">
       <property name="text">
        <string>8.8.8.8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dnsQuadMain">
       <property name="text">
        <string>9.9.9.9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dnsFreeMain">
       <property name="text">
        <string>37.235.1.174</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dnsLevelMain">
       <property name="text">
        <string>209.244.0.3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="grpSubDNS">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>171</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Main DNS Server</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="dnsGoogleSub">
       <property name="text">
        <string>8.8.4.4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dnsQuadSub">
       <property name="text">
        <string>149.112.112.112</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dnsFreeSub">
       <property name="text">
        <string>37.235.1.177</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dnsLevelSub">
       <property name="text">
        <string>209.244.0.4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="grpDomain">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>521</width>
      <height>64</height>
     </rect>
    </property>
    <property name="title">
     <string>Domain</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineDomain"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="grpCycle">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>521</width>
      <height>59</height>
     </rect>
    </property>
    <property name="title">
     <string>Cycle</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="cycleHour">
       <property name="text">
        <string>1 Hour</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="cycleMinute">
       <property name="text">
        <string>1 Minute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="cycleSecond">
       <property name="text">
        <string>1 Second</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="grpCSV">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>521</width>
      <height>64</height>
     </rect>
    </property>
    <property name="title">
     <string>CSV Path</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineCSV"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnCSV">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="grpOperation">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>521</width>
      <height>62</height>
     </rect>
    </property>
    <property name="title">
     <string>Operation</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRestart">
       <property name="text">
        <string>Restart</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
